### PR TITLE
Added objects 'Has' and 'Keys' function

### PR DESCRIPTION
**Pull request by 3timeslazy**

### CODE REVIEW

File: pkg/stdlib/objects/has.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	err = core.ValidateType(args[1], core.StringType)
+
+	if err != nil {
+		return values.None, err
+	}
+
+	obj := args[0].(*values.Object)
+	keyName := args[1].(values.String)
+
+	_, has := obj.Get(keyName)
```
Comment (ziflex) [inline pkg/stdlib/objects/has.go]: Let's change return value of ``Object.Get`` to ``(core.Value, values.Boolean)`` for consistency.

**Comment by 3timeslazy** [comment]
@ziflex The Keys method got into a commit. Remove it from the commit and add it as other PR along with the tests or leave it as is?

**Comment by ziflex** [comment]
@3timeslazy leave it as it is, just add unit tests, please :)

File: pkg/stdlib/objects/has.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+	return values.NewBoolean(bool(has)), nil
```
Comment (ziflex) [inline pkg/stdlib/objects/has.go]: After your change in ``Object`` struct, you can just return ``has`` variable now :)

Comment (3timeslazy) [reply pkg/stdlib/objects/has.go]: I just used 3th statement
https://github.com/MontFerret/ferret/issues/10#issuecomment-427376533

**Pull request merged**